### PR TITLE
[MI-1628] Implement Deploy Template

### PR DESCRIPTION
**Pull request by jmramos02**
/cc @zendesk/mintegrations

### Description
Add `DEPLOY.md` template. 

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-1628

### Risks
* [Very low] will not affecting anything in terms of app functionality

### CODE REVIEW

**Comment by jmramos02** [comment]
@z3n-badam @zendesk/sustaining

File: DEPLOY.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Deployment Process
+
+Staging and Master is automatically deployed whenever a PR is merged. Production on the other hand, have to be deployed manually and should always have a deploy buddy using samson
+
+### Staging
+
+1. Deploys automatically after a PR is merged
+2. Wait until the deploy is done successfully
+3. Test the integration using a staging account
```
Comment (jkramer0320) [inline DEPLOY.md]: can we get a link to a staging account that is set up to test changes to Shopify? Thanks!

Comment (jmramos02) [inline DEPLOY.md]: i forgot to mention that our integration is an app, so we use our own zendesk staging account for testing

Comment (jkramer0320) [inline DEPLOY.md]: Good enough for me!

**Pull request merged**